src/modules/video.ts

```diff
@@ -1,5 +1,6 @@
 import { waitForElementToLoad } from "./utils";
 import { ClientSocketHandler } from "./socket";
+import { updateStatusPanel } from "./popup";
 
 export interface EventInfo {
```

Comment:
I think we should export interfaces in some other file: `types.ts` for example

Reply:
Good idea. I was also thinking about it